packages/ui/src/components/TextArea.svelte

```diff
@@ -17,16 +17,24 @@
   import type { IntlString } from '@anticrm/platform'
   import Label from './Label.svelte'
 
-  export let label: IntlString | undefined
-  export let width: string | undefined
-  export let height: string | undefined
-  export let value: string | undefined
-  export let placeholder: string | undefined
+  export let label: IntlString | undefined = undefined
+  export let width: string | undefined = undefined
+  export let height: string | undefined = undefined
+  export let value: string | undefined = undefined
+  export let placeholder: string | undefined = undefined
+  export let onKeydown: ((e: any) => void) | undefined = undefined
```

Comment:
Please use svelte createEventDispatcher and dispatch required events.

Reply:
Done.